### ISSUE TITLE
Error listing namespaces in Visual Studio

### ISSUE THREAD

**Issue by dahovey** [microsoft/mindaro#64]
**Describe the bug**
When trying to create profile within Visual Studio no namespaces are listed and an red X is shown to right of namespace dropdown, with the tooltip "No Kubernetes cluster found. Please check your kubeconfig."
 
![image](https://user-images.githubusercontent.com/22330242/96917486-d00da000-145d-11eb-8d60-d268ad891493.png)

**To Reproduce**
I created brand new AKS Cluster, verison 1.17.11. First I had AKS cluster version 1.18.8 which also didn't work. I can successfully list namespaces using kubectl from command prompt.

 
**Expected behavior**
List the namespaces

**Logs**

[bridge-mindarocli-2020-10-22-18-57-11-46272.txt](https://github.com/microsoft/mindaro/files/5425122/bridge-mindarocli-2020-10-22-18-57-11-46272.txt)
[bridge-library-2020-10-22-18-57-12-46272.txt](https://github.com/microsoft/mindaro/files/5425123/bridge-library-2020-10-22-18-57-12-46272.txt)
[2020-10-22T18-12-05.1546952.log](https://github.com/microsoft/mindaro/files/5425124/2020-10-22T18-12-05.1546952.log)

**Environment Details**
Client used (VS Code/Visual Studio): Visual Studio
 
Client's version:
Operating System: Windows 10 Pro

**Comment by Sharmanand** [comment]
Facing same issue

**Comment by Sharmanand** [comment]
Tried issue#57 resolution and able to list namespace. https://github.com/microsoft/mindaro/issues/57

**Comment by pragyamehta** [comment]
@Sharmanand  thanks for trying the workaround - happy to know it works for you!
@dahovey Please try the workaround here #26  and let us know if it works for you.

**Comment by dahovey** [comment]
In my case I realized I had:

1. Tried using Bridge to Kubernetes on cluster that still had Azure Dev Spaces enabled.
2. Deleted k8s cluster.
3. Re-created k8s cluster with the same name.
4. Tried using Bridge to Kubernetes with new k8s cluster under the same name.

I just went ahead and deleted the cache folder in `%LOCALAPPDATA%\BridgeToKubernetes`. Then it started working as expected. From what I expect the cache should have been invalidated somehow.
